docs/resources/vpc_peering.md

```diff
@@ -6,7 +6,7 @@ description: |-
 ---
 
 ### Resource: `instaclustr_vpc_peering`  
-A resource for managing VPC peering connections on Instaclustr Managed Platform. This is only avaliable for clusters hosted with the AWS provider.  
+A resource for managing VPC peering connections on Instaclustr Managed Platform. This is avaliable for clusters hosted with the AWS and GCP .
```

Comment:
Replace `This is avaliable for clusters hosted with the AWS and GCP .` with `This is avaliable for clusters hosted with AWS and GCP.`